packages/account/src/Components/api-token/api-token-card.tsx

```diff
@@ -1,23 +1,16 @@
 import React from 'react';
-import { Field, FieldProps } from 'formik';
+import { Field, FieldProps, useFormikContext } from 'formik';
 import { CompositeCheckbox } from '@deriv/components';
 
 type TApiTokenCard = {
     description: string;
     display_name: string;
     name: string;
     value: boolean;
-    setFieldValue: (name: string, value: boolean) => void;
 };
 
-const ApiTokenCard = ({
-    name,
-    value,
-    display_name,
-    description,
-    setFieldValue,
-    children,
-}: React.PropsWithChildren<TApiTokenCard>) => {
+const ApiTokenCard = ({ name, value, display_name, description, children }: React.PropsWithChildren<TApiTokenCard>) => {
+    const { setFieldValue } = useFormikContext();
```

Comment:
As we're using `Field` from `formik`, we can get `setFieldValue` from it as well. So we can remove this FormikContext

Reply:
Ref: https://formik.org/docs/api/field